.github/workflows/nightly-macos-test.yml

```diff
@@ -6,45 +6,45 @@ on:
   
 jobs:
   common-tests:
-    runs-on: macos-13
+    runs-on: macos-latest
```

Comment:
Removing `macos-13` means we won't provide x64 macOS binaries with our releases any more. Will `dfx` cope?

Reply:
Oh, there may be another way: See https://github.com/dfinity/sdk/pull/4379/commits/41fe9b142a04629d43616fb0e4edcb09ed7317ee

Reply:
There's 2 things:
1. this nighlty job just tests macos-13, doesn't have much to do with the release; we can still do that
2. macos-13 will be completely unsupported/deprecated by github, so we won't be able to offer it anyhow at some point in the near future